go.mod

```diff
@@ -10,7 +10,7 @@ replace (
 	go.opentelemetry.io/otel/sdk => go.opentelemetry.io/otel/sdk v0.20.0
 	go.opentelemetry.io/otel/trace => go.opentelemetry.io/otel/trace v0.20.0
 	go.opentelemetry.io/proto/otlp => go.opentelemetry.io/proto/otlp v0.7.0
-	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20211202192323-5770296d904e // CVE-2021-43565
+	golang.org/x/crypto/tls => golang.org/x/crypto/tls v0.0.0-20211202192323-5770296d904e // CVE-2021-43565
```

Comment:
isn't the vuln on `ssh` package? https://groups.google.com/g/golang-announce/c/2AR1sKiM-Qs/m/9LAF9FxvBwAJ
https://go-review.googlesource.com/c/crypto/+/368814/